### PR TITLE
Support maxBindGroupsPlusVertexBuffers WebGPU limit

### PR DESCRIPTION
**Pull request by beaufortfrancois**

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@kainino0x Can you review?

File: src/library_webgpu.js [hunk 1]
```diff
@@ -725,6 +725,7 @@ var LibraryWebGPU = {
     setLimitValueU32('maxTextureDimension3D', {{{ C_STRUCTS.WGPULimits.maxTextureDimension3D }}});
     setLimitValueU32('maxTextureArrayLayers', {{{ C_STRUCTS.WGPULimits.maxTextureArrayLayers }}});
     setLimitValueU32('maxBindGroups', {{{ C_STRUCTS.WGPULimits.maxBindGroups }}});
+    setLimitValueU32('maxBindGroupsPlusVertexBuffers', {{{ C_STRUCTS.WGPULimits.maxBindGroupsPlusVertexBuffers }}});
```
Comment (kainino0x) [inline src/library_webgpu.js:728]: Also needs to be added on line 2470 (next to the other instance of `maxBindGroups`

Comment (beaufortfrancois) [reply src/library_webgpu.js:728]: You're absolutely right. I missed it somehow. Thanks for catching!

**Review by kainino0x — APPROVED** [review]

**Comment by floooh** [comment]
Linkinking this ticket to a duplicate I wrote a while ago: https://github.com/emscripten-core/emscripten/issues/20278

**Pull request merged**